app/views/components/selectItemsTable.scala.html

```diff
@@ -20,37 +20,51 @@
 
 @(table: Table)
 
-<table class="govuk-table hmrc-responsive-table" role="table">
-    <thead class="govuk-table__head" role="rowgroup">
-        <tr class="govuk-table__row" role="row">
+<table class="govuk-table hmrc-responsive-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
             @table.head.map { header =>
                 @header.map { col =>
                     <th scope="col" class="govuk-table__header @col.classes">@col.content.asHtml</th>
                 }
             }
         </tr>
     </thead>
+
     <tbody class="govuk-table__body">
+        @table.rows.map { row =>
+            <tr class="govuk-table__row" role="row">
+                @row.zipWithIndex.map { case (r, idx) =>
+                    @if(table.firstCellIsHeader && idx == 0) {
+                        <td class="govuk-table__header white-space-nowrap @r.classes" role="cell">
+                        @r.content.asHtml
+                        </td>
+                    } else {
+                        <td class="govuk-table__cell @r.classes" role="cell">
+                        @r.content.asHtml
+                        </td>
+                    }
+                }
+            </tr>
+        }
+    </tbody>
+</table>
 
+<dl class="govuk-summary-list hmrc-responsive-list">
     @table.rows.map { row =>
         <tr class="govuk-table__row" role="row">
```

Comment:
should this `<tr>` be here?

Reply:
No it shouldn't, removed with another commit